airflow/providers/google/cloud/hooks/gcs.py

```diff
@@ -185,6 +185,10 @@ def download(self, bucket_name, object_name, filename=None):
         :param filename: If set, a local file path where the file should be written to.
         :type filename: str
         """
+
+        # TODO: future improvement check file size before downloading,
```

Comment:
wdyt about creating ticket in github ticket with label `gcs todo` or `technical debt` and store there such `todos`. I am afraid it can be missed in the future.

Reply:
Let's wait for some process. FYI always before workshops, I check TODOs in code to find issues :)